application/F3DStarter.cxx

```diff
@@ -704,6 +706,7 @@ class F3DStarter::F3DInternals
       interactor.addBinding({ mod_t::NONE, "Drop" }, "add_files_or_set_hdri", "Others", std::bind(docString, "Load dropped files, folder or HDRI"));
       interactor.addBinding({ mod_t::CTRL, "Drop" }, "add_files", "Others", std::bind(docString, "Load dropped files or folder"));
       interactor.addBinding({ mod_t::SHIFT, "Drop" }, "set_hdri", "Others", std::bind(docString, "Set HDRI and use it"));
+      interactor.addBinding( { mod_t::CTRL, "O" }, "open_file", "Others", std::bind(docString, "Open File"));
```

Comment:
This will add the documentation to the cheatsheet, so related tests will fail. You need to update the baselines.

Reply:
you can download the new baselines for the CI results directly

Reply:
I've been poking around the CI artifacts but I am unsure where to find the new baselines. Also when I check the logs it says the tests pass. I'm unsure if I'm supposed to do something here or if this has already been resolved?

Reply:
CI is busted right now, lets focus on the code itself for now, we will fix the tests before the merge anyway :)